config_test.go

```diff
@@ -606,15 +606,6 @@ func TestNewQemuHypervisorConfig(t *testing.T) {
 
 }
 
-func TestNewHyperstartAgentConfig(t *testing.T) {
```

Comment:
If this is cruft we no longer need, again, could you remove this on a separate commit to make that clear?